CMakeLists.txt

```diff
@@ -127,16 +128,23 @@ set(CLP_FFI_PY_LIB_IR_SOURCES
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyExceptionContext.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyObjectCast.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/PyObjectUtils.hpp
-    ${CLP_FFI_PY_LIB_SRC_DIR}/Python.hpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/utils.cpp
     ${CLP_FFI_PY_LIB_SRC_DIR}/utils.hpp
 )
 
+# NOTE: these headers are wrappers of third-party dependency libraries. They are required to silent
+# clang-tidy warnings for missing direct includes.
+set(CLP_FFI_PY_WRAPPED_DEPENDENCY_HEADERS_SOURCES
```

Comment:
Do we need the `_SOURCES`? I feel like `CLP_FFI_PY_WRAPPED_DEPENDENCY_HEADERS` is descriptive enough and `HEADERS_SOURCES` kind of makes it sound like these should be cpp files corresponding to the headers.